### ISSUE TITLE
[BUG] Nearby nether wart causes other crops to give XP when harvested

### ISSUE THREAD

**Issue by Eksekk** [Rebirth-of-the-Night/Rebirth-Of-The-Night#317]
ISSUES NOT USING OUR TEMPLATE WILL NOT BE CONSIDERED AND WILL BE CLOSED!

**Describe the bug**
In title. [OP's description](https://discord.com/channels/620840315498004480/664351929505415168/708265267624411197): "I think I found a bug. After I placed nether warts next to my other crops, they all give me ridiculous amounts of xp when I farm them. Like lettuce is giving me xp etc".

**To Reproduce**
1. Create a farm with nether wart and other crops interspersed.
2. Harvest other crops.

**Expected behavior**
Nether wart shouldn't cause crops to drop XP.

**Screenshots**
N/A

**Info (please complete the following information):**
 - Modpack Version: unknown
 - Server or Singleplayer: Server
 - Version the world was created on: unknown

**Additional context**
N/A

**Comment by kil0byt3** [comment]
just checking, but this wasn't during a harvest moon right

**Comment by Eksekk** [comment]
Probably not, besides, even harvest moon doesn't add experience for harvesting, unless I'm wrong?

**Comment by KaleThyself** [comment]
The Harvest Moon event does not add experience for harvesting as I just now tested against lettuce and nether wart being together in a farm area. During any day or night, there was no experience gain from harvesting crops. 

Perhaps there was another mod influencing experience gain?

**Comment by democat3457** [comment]
Hol' up. (This is completely unrelated to the actual bug report.)
The OP's pfp (toddles) is the exact same one as the one I used a few years back for my email account. Spooky. (The penguin pfp)

Anyway, this bug was also experienced on a version of the pack older or equal to 2.77.1, so the bugs could have been fixed by the many intermittent changes. Also, this bug could be somewhat related to the Learning effect given by diamond armor, as suggested by #36
